### ISSUE TITLE
Sign safeguard-ps to prevent warnings

### ISSUE THREAD

**Issue by holger-weihe** [OneIdentity/safeguard-ps#359]
the safeguard-ps package is not signed so there will be a warning message when trying to install. Please sign with trusted key to prevent  the warning message.

**Comment by petrsnd** [comment]
@holger-weihe Can you test to see if this works for you?
https://www.powershellgallery.com/packages/safeguard-ps/6.12.41226-pre

**Comment by petrsnd** [comment]
Fixed in 6.12

The standard for signing PowerShell modules is not to sign the NuGet package that gets published to PowerShellGallery.org.  Instead, you sign a catalog of the contents and publish that with your module: https://docs.microsoft.com/en-us/powershell/scripting/gallery/concepts/publishing-guidelines?view=powershell-7.2#sign-your-code

You can verify the signature is correct using the following command:
```
PS> Test-FileCatalog -CatalogFilePath "C:\Program Files\WindowsPowerShell\Modules\safeguard-ps\6.12.41226\safeguard-ps.cat" -Detailed -FilesToSkip PSGetModuleInfo.xml
```

Note that `-FilesToSkip PSGetModuleInfo.xml` must be specified to work around a problem caused by v2.0 of PowerShellGet.  See here: https://github.com/PowerShell/PowerShellGet/issues/286
